### PR TITLE
adding references

### PR DESCRIPTION
**Pull request by LippkauFelix**
please review.

### CODE REVIEW

File: docs/literature/references.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## References
+
+This documentations was done by using the listet references.
```
Comment (LippkauFelix) [inline docs/literature/references.md:3]:
```suggestion
This documentation was done by using the listet references.
```

**Review by DeepakIER — CHANGES_REQUESTED** [review]
See my comment about the model overview

File: docs/index.rst [hunk 2]
```diff
@@ -17,6 +17,7 @@ Welcome to ETSAP-TIAM's documentation project!
    scenarios/index
    exogenous/index
    optional/index
```
Comment (LippkauFelix) [inline docs/index.rst:19]:
```suggestion

```

File: docs/literature/index.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# Model Overview
```
Comment (LippkauFelix) [inline docs/literature/index.md:1]:
```suggestion
# References
```

File: docs/index.rst [hunk 4]
```diff
@@ -14,9 +14,7 @@ Welcome to ETSAP-TIAM's documentation project!
    regional_coverage/index
    sectoral_coverage/index
    restrictions/index
```
Comment (LippkauFelix) [inline docs/index.rst:16]:
```suggestion
   restrictions/index
   scenarios/index
   exogenous/index
   optional/index
```

File: docs/literature/index.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (LippkauFelix) [reply docs/literature/index.md]: thanks! Changed

**Review by DeepakIER — APPROVED** [review]

**Pull request merged**